prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -58,9 +69,9 @@ class RouterCli : CliktCommand(
         if (outputDir == null && outputFileName == null) return
         reports.forEach { (report, format) ->
             val outputFile = if (outputFileName != null) {
-                File(outputFileName)
+                File(outputFileName!!)
             } else {
```

Comment:
I added this !!. It removes a warning, but it shouldn't strictly be necessary, but the compiler isn't picking up this smart cast.